### ISSUE TITLE
[Feature]: Random sorting to be added in the sorting menu

### ISSUE THREAD

**Issue by acress1** [fast4x/RiMusic#3425]
### Description

Hey ! To be able to distinguish these two buttons, it would be better to integrate the 'random sorting' into the sorting menu (via the sorting button, where we currently can choose by title, added date, etc.) 
Because at the moment these buttons are too close to each other and they look a lot alike.

Thanks!
![IMG_20240826_080834_040](https://github.com/user-attachments/assets/f11df41e-4b7f-437a-a97f-cbca25f38422)

### Suggested Solution

integrate the 'random sorting' into the sorting menu (via the sorting button, where we currently can choose by title, added date, etc.)

**Comment by aneesh1122** [comment]
It should be removed imo because if you want to randomize songs order you can always shuffle.

**Comment by acress1** [comment]
I'm just saying that the two logos are confusing because at first sight, they look very similar.

**Comment by aneesh1122** [comment]
> I'm just saying that the two logos are confusing because at first sight, they look very similar. 

So just remove the 2nd logo and use shuffle if you want to randomize songs. That makes sense. Right?

So request for the removal of 2nd logo

**Comment by acress1** [comment]
Yeah that's kinda what I would like, but to transform it in the sorting menu (up and down arrow)

**Comment by fast4x** [comment]
If enabled, auto shuffle work only one time when open songs list.
